isotovideo

```diff
@@ -435,6 +440,8 @@ while ($loop) {
         if ($rsp->{cmd} eq 'version') {
             my $result = {test_git_hash => $test_git_hash, version => $INTERFACE};
             myjsonrpc::send_json($r, $result);
+            $result = {needles_git_hash => $needles_git_hash, version => $INTERFACE};
+            myjsonrpc::send_json($r, $result);
```

Comment:
ok, so should the whole command be replaced by 'version_tests' and 'version_needles'?